src/amoc_api_status_handler.erl

```diff
@@ -26,26 +26,26 @@ trails() ->
     },
     [trails:trail("/status", amoc_api_status_handler, [], Metadata)].
 
--spec init(tuple(), cowboy:req(), state()) -> {upgrade, protocol, cowboy_rest}.
+-spec init(tuple(), cowboy_req:req(), state()) -> {upgrade, protocol, cowboy_rest}.
```

Comment:
Line 29 is too long: -spec init(tuple(), cowboy_req:req(), state()) -> {upgrade, protocol, cowboy_rest}..